security-center/snippets/v2/listAllBigQueryExports.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ *  List BigQuery exports in the given parent.
+ */
+function main(organizationId, location = 'global') {
+  // [START securitycenter_list_bigquery_export_v2]
+  // Imports the Google Cloud client library.
+  const {SecurityCenterClient} = require('@google-cloud/security-center').v2;
+
+  // Creates a new client.
+  const client = new SecurityCenterClient();
+  /**
+   *  Required. The parent, which owns the collection of BigQuery exports. Its
+   *  format is "organizations/[organization_id]/locations/[location_id]",
+   *  "folders/[folder_id]/locations/[location_id]", or
+   *  "projects/[project_id]/locations/[location_id]".
+   */
+  const parent = client.organizationLocationPath(organizationId, location);
```

Comment:
issue: Referencing organizationId and location from outside the region tag scope is a violation of the copy-paste-runnable goal for samples.

suggestion: Minimally, any referenced variable should have a commented out variable declaration with an instruction for use on each variable.

For example,

```
  /**
   * TODO(developer): Uncomment the following line before running the sample.
   */

  // const projectId = 'YOUR_PROJECT_ID';
```

In https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/security-center/snippets/v2/createFinding.js it appears the decision was made to hard-code location to global. If that's how we're testing, hardcoding the global value without comment and adding an inline comment explaining would be better than a commented out instantiation.

See #3830 for previously discussed instances

Reply:
Addressed.